mods/ra/rules/aircraft.yaml

```diff
@@ -340,43 +340,46 @@ HIND:
 	Inherits@AUTOTARGET: ^AutoTargetGroundAssaultMove
 	Buildable:
 		Queue: Aircraft
-		BuildAtProductionType: Helicopter
-		BuildPaletteOrder: 20
-		Prerequisites: ~hpad, ~techlevel.medium
-		Description: Helicopter gunship armed\nwith dual chainguns.\n  Strong vs Infantry, Light armor\n  Weak vs Tanks, Aircraft
+		BuildAtProductionType: Plane
+        BuildPaletteOrder: 35
+		Prerequisites: ~afld, ~techlevel.medium
+		Description: Fast infantry transport helicopter\n Armed with dual chainguns\n Strong vs Infantry.\n Weak vs Tanks, Aircraft
```

Comment:
Second sentence shouldn't start with a space, also either there should be a dot after "helicopter" or "Armed" should be "armed". The dot after "Infantry" should go back to after "chainguns".

Reply:
fixed